pipes/01-mp-pipe.py

```diff
@@ -1,14 +1,12 @@
+import os.path
 import xml.etree.ElementTree as ET
 import csv
 import psycopg2
 from harmonize import harmonize_party
 
-mp_data = []
+with open(os.path.join("data", "MemberOfParliament.tsv")) as f:
 
-with open("MemberOfParliament.tsv") as fd:
-
-    for row in csv.reader(fd, delimiter="\t", quotechar='"'):
-        mp_data.append(row)
+    mp_data = [row for row in csv.reader(f, delimiter="\t", quotechar='"')]
```

Comment:
jos kiinnostaa tarpeeks autismoida ni `[*csv.reader(f, ...)]` taitaa olla laskentatehokkain tapa tähän, ehk ite sanoisin et list(csv.reader(f, ...))` on parempi iha vaa luettavuuden takii 

ts. ei välttis oo valuee tehä `a for a in iterator` jos ei haluu transformata `a`

ref: https://stackoverflow.com/questions/3790848/how-can-i-create-a-list-of-elements-from-an-iterator-convert-the-iterator-to-a

Reply:
sama pätee kaikis muiski kohis